src/borg/archiver.py

```diff
@@ -922,8 +922,8 @@ def _export_tar(self, args, archive, tar):
         hardlink_masters = {} if partial_extract else None
 
         def peek_and_store_hardlink_masters(item, matched):
-            if (partial_extract and not matched and hardlinkable(item.mode) and
-                    item.get('hardlink_master', True) and 'source' not in item):
+            if ((partial_extract and not matched and hardlinkable(item.mode)) and
+                    (item.get('hardlink_master', True) and 'source' not in item)):
```

Comment:
Since these parentheses don't change the semantics of this expression, what do they convey?

Reply:
no they don't, but in borg extract there is some explanation explaining the 2 toplevel terms.
i just copied same parens structure from there.